data/Makefile

```diff
@@ -642,11 +642,35 @@ minsize=2000
 
 # Index a FASTA file.
 %.k$k-w$w.physlr.tsv: %.fa
+ifeq ($(r),)
+ifeq ($(s),)
```

Comment:
Is there a default value for `r` and `s` in the Makefile? I'm thinking if at least there is a default setting we can add a comment to know what that variable means

Reply:
I've added a comment and made the variables more verbose so people should understand the intention